core/search/search.py

```diff
@@ -62,11 +78,15 @@ def visit_phrase(self, node: Phrase, parents: List[Item], context: SQLQueryBuild
         return self.visit_term(node, parents, context)
 
     def visit_range(self, node: Range, parents: List[Item], context: SQLQueryBuilderContext) -> Range:
+
         """
         Visitor for Range
         - inclusive [<Term> TO <Term>]
         - exclusive [<Term> TO <Term>]
```

Comment:
I understand that it's unrelated but that helper comment is wrong what can be fixed in this PR.
```suggestion
        - exclusive {<Term> TO <Term>}
```